lib/cli.js

```diff
@@ -42,5 +49,12 @@ module.exports = function() {
     });
   });
 
+  process.on('uncaughtException', function(error) {
```

Comment:
Is this related to the clean cached accessories?

Reply:
Not really, sorry.  When I commit a new change, GitHub adds this to the open PR.

It’s actually more related to the check for the number of bridged accessories in HAP-NodeJS, since that error is unchecked by homebridge.  I’ve had this logic in my homebridge-zp plugin for months, to make sure the subscriptions with the Sonos speakers are canceled before homebridge exits.  Only then, the error is logged by the plugin logger, making it seem like my plugin caused the exception instead of only handling it.  See https://github.com/ebaauw/homebridge-zp/issues/41 and https://github.com/ebaauw/homebridge-zp/issues/49.